bin/coqtop.ml

```diff
@@ -64,6 +66,12 @@ let term =
         let* coqtop, args =
           let open Memo.O in
           Build_system.run_exn (fun () ->
+              let* tr = Dune_rules.Dir_contents.triage sctx ~dir in
+              let dir =
+                match tr with
+                | Dune_rules.Dir_contents.Group_part dir -> dir
+                | Dune_rules.Dir_contents.Standalone_or_root _ -> dir
```

Comment:
could you annotate the type of `tr` instead of qualifying the constructors?

That's the style in the rest of dune.

Reply:
Sure, done.